tests/system/src/test/java/test/robot/scenegraph/JDK8130122Test.java

```diff
@@ -98,9 +98,10 @@ public void testEmptyShapes() {
             assertColorEquals(Color.WHITE, color, TOLERANCE);
             horizontalListView.setVisible(true);
         });
-        waitNextFrame();
+        // Give more time after setVisible(true) is called for frame update
+        waitFirstFrame();
```

Comment:
this is fine, though I'd rather have a dedicated method - waitFirstFrame() implies that it should only be used to wait for the first frame, that it might have a different purpose and implementation.